### PR TITLE
Add post start task to etcd job to check consistency

### PR DESCRIPTION
**Pull request by mkjelland**
[CI pipeline](https://ci.kubo.sh/teams/main/pipelines/etcd-mkjelland) for this branch

[Story in tracker](https://www.pivotaltracker.com/story/show/150781092)

### CODE REVIEW

File: jobs/etcd/templates/post-start.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  set +e
+  local consistency_check_output
+  consistency_check_output=$(timeout 10 "${PKG_DIR}/bin/etcd-consistency-checker" \
+    --cluster-members "${consistency_checker_cluster_members}" \
+    ${tls_flags})
+  exit_code="$?"
+  set -e
+
+  if [ "124" != "$exit_code" ] && [ "0" != "$exit_code" ]; then
```
Comment (alex-slynko) [inline jobs/etcd/templates/post-start.erb:40]: Why is timeout ok here?

Comment (mkjelland) [reply jobs/etcd/templates/post-start.erb:40]: The consistency checker runs on a constant loop if it succeeds

**Comment by jhvhs** [comment]
Hi Meaghan,
Why do we need the post-start script? If we introduce a timeout to the consistency checker, it is going to fail the deployment straight away before the post-start script gets a chance to run.

**Comment by mkjelland** [comment]
@jhvhs it didn't seem like it was running as part of the deploy process. If that works then it seems like a better solution. This pr predates a lot of the conversions we've been having about etcd and may no longer be relevant

**Comment by mkjelland** [comment]
Closing in favor of #2

**Pull request closed without merging**